src/biokbase/narrative/tests/generate_test_results.py

```diff
@@ -0,0 +1,219 @@
+import copy
```

Comment:
Rather than generating the expected messages in numerous places (`test_job.py`, `test_jobcomm.py`, `test_jobmanager.py`), it is all done in this file -- much easier to amend if required.

Reply:
And better to avoid cross-test dependencies.